cetmix_tower_server/models/cx_tower_file_template.py

```
@@ -136,45 +136,71 @@ def create_file(self, server, server_dir="", raise_if_exists=False):
         :param server: recordset
             The server (cx.tower.server) on which the file should be created. This is a
             required parameter.
+        :param if_file_exists: str, optional
+            Defines the behavior if the file already exists on the server.
         :param server_dir: str, optional
             The directory on the server where the file should be created. If not set,
```

Flake8 (7.2.0)

[error] line too long (85 > 79 characters)

(E501)